.github/workflows/arc-deploy-prod.yml

```diff
@@ -40,3 +40,4 @@ jobs:
         AWS_DEFAULT_REGION: us-east-1
         GITHUB_TOKEN: ${{ secrets.LIST_PYTORCH_RUNNERS_GITHUB_TOKEN }}
         TERRAFORM_EXTRAS: -auto-approve -lock-timeout=15m
+        TF_VAR_cert_manager_email: ${{ secrets.CERT_MANAGER_EMAIL }}
```

Comment:
This secret needs to be created on GitHub and include the email address used for cert-manager `ClusterIssuer`. That email address is sent to Let's Encrypt when the certificate is requested, but it's not associated with an account, nor is it stored in the certificate itself. 

Let's Encrypt used to send notification emails related to certificate expirations, but it no longer does.
The cert-manager component on the cluster takes care of renewing the certificate when needed.

Reply:
If you don't mind storing the email address in git, it can be added as a default value to `variables.tf` and then this environment variable and the need for a secret can be removed

Reply:
I think it should be okay to store it in Git. Emails aren't really "secret" anyway. You can use this email address `hostmaster+pytorch-ci-argocd@linuxfoundation.org`.

@jordanconway and I should get any emails that come in on that if we need to check.